packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -17,11 +21,53 @@ class AdaptiveCard extends React.Component {
   componentDidMount() {
     if (Object.prototype.hasOwnProperty.call(this, 'nodeElement') && Object.prototype.hasOwnProperty.call(this.nodeElement, 'current')) {
       this.nodeElement.current.appendChild(
-        getAdaptiveCard(this.props.cards, this.props.displayName, this.props.sparkInstance, this.addChildNode)
+        getAdaptiveCard(this.state.cards, this.props.displayName, this.props.sparkInstance, this.addChildNode)
       );
     }
   }
 
+  static getDerivedStateFromProps(nextProps) {
+    if (nextProps.verb === API_ACTIVITY_VERB.SHARE) {
+      try {
+        const decryptedURLs = nextProps.items
+          .filter((file) => file.type === CARD_CONTAINS_IMAGE)
+          .map((file) => {
+            const thumbnail = file.mimeType === 'image/gif' ? nextProps.share.getIn(['files', file.url]) : nextProps.share.getIn(['files', file.image.url]);
+
+            if (thumbnail) {
```

Comment:
Where would it be a case where there is no `thumbnail`?

Reply:
This condition is for avoiding the error if trying to get objectURL when there is no content in `thumbnail`